src/charm.py

```diff
@@ -211,8 +211,6 @@ def _is_valid_timespec(self, timeval) -> bool:
         Returns:
             True if time specification is valid and False otherwise.
         """
-        matched = re.match(r"[1-9][0-9]*[ymwdhs]", timeval) is not None
-
         if not (matched := re.match(r"[1-9][0-9]*[ymwdhs]", timeval)):
```

Comment:
tackle #128 while you're at it?

Reply:
I did think this through. The docs are not right. Try intervals starting with zero. For instance "0s". In any case I do not want to allow such intervals. I will be closing that ticket for the same reason.

Reply:
The existing regex does not support expression such as `1h30m` though.
It is odd however to silently ignore user input in the charm (e.g `0`) that is permissible by official docs.

Reply:
you mean prom supports expressions such as `3600s` and `90m`?
anyway, this is a minor thing and I think it would be best to be consistent with prom.

Reply:
Yes. Exactly.